packages/ember-testing/lib/helpers/find.js

```diff
@@ -29,6 +30,15 @@ import { get } from 'ember-metal';
 export default function find(app, selector, context) {
   let $el;
   context = context || get(app, 'rootElement');
-  $el = app.$(selector, context);
-  return $el;
+  if (jQueryDisabled) {
+    let rootElement = document.querySelector(context);
+    $el = rootElement.querySelector(selector);
```

Comment:
This should be returned, (we don't need to wrap in an array)

Reply:
Also, I think we should do:

```js
  let result;
  if (context instanceof Element) {
    result = context.querySelector(selector);
  } else if (typeof context === 'string') {
    result = document.querySelector(`${context} ${selector}`);
  } else {
    result = document.querySelector(selector);
  }
  return result;
```

Reply:
Fixed! :taco:   

So to support both jquery and JS, I think we need to return an array or array-like object from find.  Other helpers are expecting this (like findWithAssert).  Unless we wanted to go with only returning one element; however, perhaps users were using `find` to select multiple as show [here](https://github.com/emberjs/ember.js/blob/master/packages/ember-testing/tests/integration_test.js#L74).